### PR TITLE
DOC: reformat install.rst testing section

### PR DESCRIPTION
**Pull request by jreback**
TST: `run.test()` now defaults warnings based on release version

### CODE REVIEW

File: doc/source/install.rst [hunk 1]
```diff
@@ -192,32 +192,30 @@ installed), make sure you have `nose
...
-    ..........................................................................
-    .................S........................................................
-    ....
+    $ import pandas as pd
+    $ pd.test()
+    Running unit tests for pandas
+    pandas version 0.18.0
+    numpy version 1.10.2
+    pandas is installed in pandas
+    Python version 2.7.11 |Continuum Analytics, Inc.| (default, Dec  6 2015, 18:57:58) [GCC 4.2.1 (Apple Inc. build 5577)]
```
Comment (jreback) [inline doc/source/install.rst]: @jorisvandenbossche do I just manually wrap this? or leave it

Comment (jorisvandenbossche) [inline doc/source/install.rst]: I would just manually wrap it, otherwise it will generate a horizontal scroll bar in the literal code block

Comment (jorisvandenbossche) [inline doc/source/install.rst]: BTW, I would change the `$` to `>>>` as this is now in terminal?

Comment (jreback) [inline doc/source/install.rst]: sure, yeh that was some *really* old code. (and originally 800 tests!)

**Comment by jreback** [comment]
merged via https://github.com/pydata/pandas/commit/83fe3f3157b766eb497bdffd7ed3f57a3706859d

**Pull request closed without merging**